site/content/en/docs/tasks/troubleshooting/troubleshooting_provreq.md

```diff
@@ -0,0 +1,165 @@
+
+```bash
+kubectl describe provisioningrequest PROVISIONING_REQUEST_NAME
+```
+
+If your ProvisioningRequest fails to provision nodes, the error output may look similar to this:
+```bash
+[...]
+Status:
+  Conditions:
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't accepted.
+    Observed Generation:   1
+    Reason:                NotAccepted
+    Status:                False
+    Type:                  Accepted
+    Last Transition Time:  2024-05-22T13:04:54Z
+    Message:               Provisioning Request wasn't provisioned.
+    Observed Generation:   1
+    Reason:                NotProvisioned
+    Status:                False
+    Type:                  Provisioned
+    Last Transition Time:  2024-05-22T13:06:49Z
+    Message:               max cluster limit reached, nodepools out of resources: default-nodepool (cpu, memory)
+    Observed Generation:   1
+    Reason:                OutOfResources
+    Status:                True
+    Type:                  Failed
+```
+
+Note that the `Reason` and `Message` values for `Failed` condition may differ from your output, depending on an error.
+
+Provisioning Request state is described in the `.conditions[*].status` field. An empty field means ProvisinongRequest is still being processed by the ClusterAutoscaler. Otherwise, it falls into one the states listed below:
+- `Accepted` - indicates that the ProvisioningRequest was accepted by ClusterAutoscaler, so ClusterAutoscaler will attempt to provision the nodes for it.
+- `Provisioned` - indicates that all of the requested resources were created and are available in the cluster. ClusterAutoscaler will set this condition when the VM creation finishes successfully.
+- `Failed` - indicates that it is impossible to obtain resources to fulfill this ProvisioningRequest. Condition Reason and Message will contain more details about what failed.
+- `BookingExpired` - indicates that the ProvisioningRequest had Provisioned condition before and capacity reservation time is expired.
+- `CapacityRevoked` - indicates that requested resources are not longer valid.
+
+The states transitions are as follow:
+
+![Provisioning Request's states](/images/prov-req-states.svg)
+
+## Why a Provisioning Request is not created?
+
+If your job does not create a corresponding Provisioning Request try checking following requirements:
+
+### Ensure the Kueue's controller manager enables the `ProvisioningACC` feature gate
+
+Run the following command to check whether your Kueue's controller manager has enabled the `ProvisioningACC` feature gate:
+
+```bash
+kubectl describe pod -n kueue-system kueue-controller-manager-
+```
+
+The args for kueue container should be similar to this:
+
+```bash
+    ...
+    Args:
+      --config=/controller_manager_config.yaml
+      --zap-log-level=2
+      --feature-gates=ProvisioningACC=true
+```
+
+### Ensure the ClusterQueue and the LocalQueue are active
```

Comment:
More generally, users need to check whether the Workload has the QuotaReserved condition. If it doesn't, then they can follow troubleshooting_jobs to understand why.

Reply:
Fixed